modules/memory.sh

```diff
@@ -70,6 +70,14 @@ function memory() {
   checkargn $# 2
   check_missing_packages "bc"
 
+  COMP=""
+  if [ "$(detectrpi)" = "nonrpi" ];
+  then 
+	  COMP="vagrant"
+  else
+	  COMP="rpi"
+  fi 
+	
   if [ "$1" == "total" ] ; then
```

Comment:
Make sure there aren't any trailing white spaces and convert the tabs into spaces.